cnv/cnv_users_guide/cnv-openshift-cluster-monitoring.adoc

```diff
@@ -9,18 +9,17 @@ toc::[]
 // Cluster monitoring
 include::modules/monitoring-about-cluster-monitoring.adoc[leveloffset=+1]
 
-== Cluster logging
-
-The cluster logging components are based upon Elasticsearch, Fluentd, and Kibana (EFK).
-The collector, link:http://www.fluentd.org/architecture[Fluentd], is deployed to each node in the {product-title} cluster.
-It collects all node and container logs and writes them to link:https://www.elastic.co/products/elasticsearch[Elasticsearch] (ES).
-link:https://www.elastic.co/guide/en/kibana/current/introduction.html[Kibana] is the centralized, web UI
-where users and administrators can create rich visualizations and dashboards with the aggregated data.
+// Cluster logging
+include::modules/cluster-logging-about.adoc[leveloffset=+1]
 
 For more information on cluster logging, see the xref:../../logging/efk-logging.adoc#efk-logging[{product-title} cluster logging] documentation.
 
 // Telemetry
 include::modules/telemetry-about-telemetry.adoc[leveloffset=+1]
+<<<<<<< HEAD
+=======
+include::modules/telemetry-what-information-is-collected.adoc[leveloffset=+2]
+>>>>>>> d46918c... adding conditionals so that OCP logging module can be re-used in CNV.
```

Comment:
@mburke5678 - would need to remove this cruft and keep the include

Reply:
@aburdenthehand Thanks! Thought I got that one. Must not have saved the file. All set!